specification/storage/Microsoft.BlobStorage/suppressions.yaml

```diff
@@ -8,4 +8,11 @@
     - options.@azure-tools/typespec-go.inject-spans
     - options.@azure-tools/typespec-go.service-dir
     - options.@azure-tools/typespec-go.package-dir
-  reason: 'Suppress Go options until service team is ready to update the go package with typespec'
\ No newline at end of file
+    - options.@azure-tools/typespec-java.emitter-output-dir
+    - options.@azure-tools/typespec-ts.emitter-output-dir
+    - options.@azure-tools/typespec-ts.package-details.name
+    - options.@azure-tools/typespec-python.emitter-output-dir
+    - options.@azure-tools/typespec-csharp.namespace
+    - options.@azure-tools/typespec-csharp.clear-output-folder
+    - options.@azure-tools/typespec-csharp.emitter-output-dir
+  reason: 'Suppress SDK configuration validation until TypeSpec conversion is ready. Go options suppressed until service team is ready to update the go package with typespec.'
```

Comment:
jfyi you could have just disabled the whole check, like in this example, but you can leave as is: 
<img width="1440" height="487" alt="image" src="https://github.com/user-attachments/assets/d2d22758-742a-45ef-b9b2-7772d9f32395" />